src/app/choose-template/page.tsx

```diff
@@ -0,0 +1,60 @@
+'use client' // we need this to be a client component because nhsuk-react-components uses client-only react features
+import { Radios, Button, ErrorSummary, ErrorMessage,  } from 'nhsuk-react-components';
+import { useFormState } from 'react-dom';
+import { chooseTemplateServerAction, FormState } from './choose-template-server-action';
+import { chooseTemplatePageContent } from '../../content/content';
+
+export default function ChooseTemplatePage() {
+    const { pageHeading, errorHeading, options } = chooseTemplatePageContent;
+    const [state, action] = useFormState<FormState, FormData>(chooseTemplateServerAction, { formErrors: [], fieldErrors: {}});
+    console.log(state);
+    console.log(Object.entries(state.fieldErrors));
+    const hasError = state.formErrors.length > 0 || Object.keys(state.fieldErrors).length > 0;
+    return (
+        <>
+            {hasError && (
+            <div>
+                <ErrorSummary>
```

Comment:
Potential candidate for a refactor (separate function or non-reusable component in the same directory).